specification/machinelearning/resource-manager/Microsoft.MachineLearning/stable/2017-01-01/webservices.json

```diff
@@ -16,6 +16,23 @@
     "application/json"
   ],
   "paths": {
+    "/providers/Microsoft.MachineLearning/operations": {
+      "get": {
+        "description": "Lists all the available REST API operations.",
```

Comment:
make this operation pageable (if `OperationEntityListResult` only has one page, set next link name to `null` as explained in other Swagger)

Reply:
Done